idaes_examples/notebooks/docs/flowsheets/hda_flowsheet_with_distillation.ipynb

```diff
@@ -293,9 +294,26 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 10,
+   "execution_count": 207,
    "metadata": {},
-   "outputs": [],
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "WARNING: Params with units must be mutable.  Converting Param\n",
```

Comment:
I assume this warning is coming for a core property package, and this cannot be fixed as part of this PR. Could you open an issue about this in IDAES Core so that we can get it fixed there?

Reply:
I shall create a PR there.

Reply:
I assume these files will be regenerated once the IDAES Core fix is merged.

Reply:
The IDAES core fix is merged now. Should I run the files again?